### PR TITLE
Update sass linting

### LINKED ISSUE #2: Update Sass linting section
As stated in the readme, we’ve switched to using [Stylelint](https://github.com/stylelint/stylelint). We should remove the older sass-lint config files & replace with our current stylelint configs.

### PR DESCRIPTION
**Pull request by planktonic**
Fixes #2.
- Removes the sasslint files in favour of our current stylelint configs
- Creates an npm package

@weiland @pmeinhardt please take a look — I’ve stuck to the format you established in the `.eslintrc` files, hopefully all is correct. Works locally via `npm link`, yet to be published.

ESLinting the js here seemed overkill, but maybe you feel otherwise?

### CODE REVIEW

**Comment by pmeinhardt** [comment]
How, cool! That will make using and also updating the config in our projects much easier 👍

File: linting/stylelint-config-bitcrowd/overrides.js [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+      "resize",
+      "appearance",
+      "user-select",
+      "interpolation-mode",
+      "direction",
+      "marks",
+      "page",
+      "set-link-source",
+      "unicode-bidi",
+      "speak",
```
Comment (pmeinhardt) [inline linting/stylelint-config-bitcrowd/overrides.js]: Maybe linting would be useful? Normally we don't add dangling commas.

Comment (pmeinhardt) [reply linting/stylelint-config-bitcrowd/overrides.js]: …but I would also be okay with just removing the dangling now and - possibly - taking care of linting in a separate issue/PR.

**Review by pmeinhardt — CHANGES_REQUESTED** [review]

File: linting/stylelint-config-bitcrowd/overrides.js [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+    ]
+  }
+}
```
Comment (pmeinhardt) [inline linting/stylelint-config-bitcrowd/overrides.js]: Also, there's a semicolon missing here.

**Review by pmeinhardt — CHANGES_REQUESTED** [review]

File: linting/stylelint-config-bitcrowd/overrides.js (hunk 1 above)
Comment (planktonic) [reply linting/stylelint-config-bitcrowd/overrides.js]: Hah! Perfect illustration that linting is indeed not overkill. On it 👍

**Comment by planktonic** [comment]
@pmeinhardt I’ve added linting (`eslint-config-bitcrowd-base`) and fixed the linting errors reported.

**Review by pmeinhardt — APPROVED** [review]

**Comment by pmeinhardt** [comment]
Cool, looks nice 😃

**Pull request merged**